### PR TITLE
Implement bundleHook method

### PR DESCRIPTION
**Pull request by lo1tuma**
This adds a new method `bundleHook` which bundles a file similar to `bundleRule` but it assigns the wrapped function to `module.exports`.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,7 +6,7 @@
 [![Build status][travis-ci-image]][travis-ci-url]
 [![License][license-image]][license-url]
 
-Bundle rules, scripts and hooks to deploy them to Auth0.
```
Comment (selaux) [inline README.md]: Scripts could link to: https://auth0.com/docs/connections/database/mysql

Comment (lo1tuma) [inline README.md]: ✅

File: lib/auth0Bundler.js [hunk 2]
```diff
@@ -12,6 +12,7 @@ const buildLiteralAst = require('./buildLiteralAst');
 const dependencies = { rollup, babelTransform, BabelPluginExportToFunction, buildLiteralAst, rollupCommonjs };
 
 module.exports = {
-    bundleScript: R.partial(bundleRule, [ dependencies ]),
-    bundleRule: R.partial(bundleRule, [ dependencies ])
+    bundleScript: R.partial(bundleRule, [ dependencies, { commonjs: false } ]),
```
Comment (selaux) [inline lib/auth0Bundler.js]: Maybe we should rename `bundleRule` as it now bundles everything: Just `bundle` would be ok imho.

Comment (lo1tuma) [inline lib/auth0Bundler.js]: Done 😉

**Pull request merged**